composer.json

```diff
@@ -40,9 +40,11 @@
     "require-dev": {
         "doctrine/coding-standard": "^8.1",
         "jetbrains/phpstorm-stubs": "^2019.1",
-        "phpstan/phpstan": "^0.12.40",
+        "phpstan/phpstan": "^0.12.54",
+        "phpstan/phpstan-phpunit": "^0.12.16",
+        "phpstan/phpstan-strict-rules": "^0.12.5",
```

Comment:
The two dependencies above should not be introduced. They are only used in the newer versions of the DBAL.